src/audio_midiout_device.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EP_AUDIO_MIDIOUT_DEVICE_H
+#define EP_AUDIO_MIDIOUT_DEVICE_H
+
+#include <cstdint>
+#include <cstddef>
+
+ /**
+  * Manages sequencing MIDI files and emitting MIDI events to a MIDI out
+  * device.
+  */
+class MidiOutDevice {
+public:
+	virtual ~MidiOutDevice() = default;
```

Comment:
Copy and move operations `= delete`

Reply:
So I admit this is butting up against the limits of my C++ knowledge. Before EasyRPG the last C++ I ever did was C++03 so half of these words are foreign to me. Despite that, I've had a red-hot go, let me know how I did

Reply:
```
MidiOutDevice& operator=(const MidiOutDevice&) = delete;
MidiOutDevice& operator=(MidiOutDevice&&) = delete;
```